modules/unsupported/pom.xml

```diff
@@ -1,237 +1,238 @@
-				<module>process</module>
-				<module>process-geometry</module>
-				<module>process-raster</module>
-				<module>process-feature</module>
-			</modules>
-		</profile>
-		<profile>
-			<id>swt</id>
-			<modules>
-				<module>swt</module>
-			</modules>
-		</profile>
-		<profile>
-			<id>feature-aggreate</id>
-			<modules>
-				<module>feature-aggreate</module>
-			</modules>
-		</profile>
-		<profile>
-			<id>solr</id>
-			<activation>
-				<property>
-					<name>all</name>
-				</property>
-			</activation>
-			<modules>
-				<module>solr</module>
-			</modules>
-		</profile>
-		<profile>
-			<id>geobuf</id>
-			<activation>
-				<property>
-					<name>all</name>
-				</property>
-			</activation>
-			<modules>
-				<module>geobuf</module>
-			</modules>
-		</profile>
-		<profile>
-			<id>css</id>
-			<activation>
-				<property>
-					<name>all</name>
-				</property>
-			</activation>
-			<modules>
-				<module>css</module>
-			</modules>
-		</profile>
-		<profile>
-			<id>geojsonstore</id>
-			<activation>
-				<property>
-					<name>all</name>
-				</property>
-			</activation>
-			<modules>
-				<module>geojsonstore</module>
-				<module>geojson</module>
-			</modules>
-		</profile>
-        <profile>
-          <id>javafx</id>
-          <modules>
-            <module>javafx</module>
-          </modules>
-        </profile>
-        <profile>
-            <id>mbstyle</id>
-            <activation>
-                <property>
-                    <name>all</name>
-                </property>
-            </activation>
-            <modules>
-                <module>mbstyle</module>
-            </modules>
-        </profile>        
-        <profile>
-            <id>s3-geotiff</id>
-            <modules>
-                <module>s3-geotiff</module>
-            </modules>
-        </profile>
-        <profile>
-            <id>ogr</id>
-            <modules>
-                <module>ogr</module>
-            </modules>
-        </profile>
-		<profile>
-			<id>unsupported</id>
-			<activation>
-				<property>
-					<name>all</name>
-				</property>
-			</activation>
-			<modules>
-				<module>csv</module>
-				<module>jdbc-ng</module>
-				<module>solr</module>
-				<module>css</module>
-				<module>geojsonstore</module>
-        		<module>s3-geotiff</module>
-        		<module>gtopo30</module>
-        		<module>vpf</module>
-			</modules>
-		</profile>
-
-		<profile>
-			<id>arcgis-rest</id>
-			<modules>
-				<module>arcgis-rest</module>
-			</modules>
-		</profile>
-
-	</profiles>
-
-	<!-- Modules included in a normal build -->
-	<modules>
-		<module>vpf</module>
-		<module>swing</module>
-		<module>process</module>
-		<module>process-geometry</module>
-		<module>process-raster</module>
-		<module>process-feature</module>
-		<module>geojson</module>
-		<module>mbtiles</module>
-		<module>wfs-ng</module>
-		<module>ogr</module>
-		<module>postgis-raster</module>
-	</modules>
-
-</project>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please don't rewrite the entire file (it's newlines I think)

Reply:
Ok this is fixed.